.vscode/tasks.json

```diff
@@ -0,0 +1,190 @@
+        },
+        {
+            "label": "Format code",
+            "command": "go fmt ./...",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Update golang dependencies",
+            "command": "go mod tidy",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Run unit tests",
+            "command": "make test",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Update OLM bundle",
+            "command": "make bundle -s",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Update CR/CRDs",
+            "command": "make generate manifests -s",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Build and push custom operator image: '${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator:next'",
+            "command": "make docker-build docker-push IMG=\"${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator:next\"",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Install OLM on the cluster",
+            "command": "operator-sdk olm install",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Build and push development bundle",
+            "command": "export BUNDLE_IMG=\"${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator-bundle:next\"; make bundle IMG=${BUNDLE_IMG} -s; make bundle-build bundle-push -s BUNDLE_IMG=${BUNDLE_IMG}",
```

Comment:
For `make bundle`, I think we should  specify the `kubernetes-image-puller-operator:next` image instead of the `kubernetes-image-puller-operator-bundle:next` image?

Reply:
Sorry, but, no, I used correct image. kubernetes-image-puller-operator-bundle:next  - that's image needed for OLM. This image - that's a storage with bundle files, including CSV file(see 'bundle/manifest' folder in this pr). In the Cluster service version(CSV) file stored operator deployment(in the deployments section) with defined operator image: 

https://github.com/che-incubator/kubernetes-image-puller-operator/blob/211f2bf87b529a2279188ae51b443a295e91c842/bundle/manifests/kubernetes-imagepuller-operator.clusterserviceversion.yaml#L104

There is rule for new bundle format: single bundle per  bundle image. So single bundle image contains only one operator version.